pkgs/servers/dns/bind/dont-keep-configure-flags.patch

```diff
@@ -2,38 +2,38 @@ diff --git a/bin/named/include/named/globals.h b/bin/named/include/named/globals
 index 82b632ef04..dedfd4d33b 100644
 --- a/bin/named/include/named/globals.h
 +++ b/bin/named/include/named/globals.h
-@@ -71,7 +71,9 @@ EXTERN const char *named_g_version	  INIT(VERSION);
- EXTERN const char *named_g_product	  INIT(PRODUCT);
- EXTERN const char *named_g_description	  INIT(DESCRIPTION);
- EXTERN const char *named_g_srcid	  INIT(SRCID);
+@@ -69,7 +69,9 @@ EXTERN const char *named_g_version	INIT(PACKAGE_VERSION);
+ EXTERN const char *named_g_product	INIT(PACKAGE_NAME);
+ EXTERN const char *named_g_description	INIT(PACKAGE_DESCRIPTION);
+ EXTERN const char *named_g_srcid	INIT(PACKAGE_SRCID);
 +#if 0
- EXTERN const char *named_g_configargs	  INIT(CONFIGARGS);
+ EXTERN const char *named_g_configargs	INIT(PACKAGE_CONFIGARGS);
 +#endif
- EXTERN const char *named_g_builder	  INIT(BUILDER);
- EXTERN in_port_t named_g_port		  INIT(0);
- EXTERN isc_dscp_t named_g_dscp		  INIT(-1);
+ EXTERN const char *named_g_builder	INIT(PACKAGE_BUILDER);
+ EXTERN in_port_t named_g_port		INIT(0);
+ EXTERN in_port_t named_g_tlsport	INIT(0);
```

Comment:
Looking at the original file, it [looks like](https://gitlab.isc.org/isc-projects/bind9/-/blob/51147fa567ce6cb301fad0e882f48f166bb8e2d8/bin/named/include/named/globals.h#L78) `named_g_tlsport` doesn't need to be patched, meaning that line should probably not be in the patch. Possibly others too.

Reply:
I think you are just a little confused because reading patches of patches is very confusing. Perhaps it would be easier to look at the full file before and after. `named_g_tlsport` isn't really "patched", it's just part of the patch. The patch itself just adds a couple lines containing `#if 0` and `#endif`. Everything else is just code from the original source above and below those two lines, which is used when applying the patch. In case the patch doesn't apply exactly, for example because the line numbers changed, the patch can still be applied this way. If I were to change `named_g_tlsport` back to `named_g_dscp` again, I'm pretty sure, the patch wouldn't apply anymore. If I'm wrong and I actually did something wrong here, please tell me again.

Reply:
You're right, I did get confused :sweat_smile: